buildpacks/nodejs-engine/tests/integration_test.rs

```diff
@@ -10,7 +10,10 @@ use test_support::{
 #[ignore]
 fn simple_indexjs() {
     nodejs_integration_test("./fixtures/node-with-indexjs", |ctx| {
-        assert_contains!(ctx.pack_stdout, "Detected Node.js version range: *");
+        assert_contains!(
+            ctx.pack_stdout,
+            "Detected Node.js version range: >=20.0.0 <21.0.0-0"
```

Comment:
This log output seems like it would be confusing to users or support. It's not that the `package.json` specified that version, but that it's the default. It seems the default case should be made to have a different log message?

Whilst this issue is pre-existing, now that the default version isn't the wildcard I think it makes it even more confusing and so important to resolve.

Reply:
Agreed. I've updated the code to branch if there is a version given in `package.json` and:
- if present, the `Detected Node.js version range: ...` message will be shown
- if not present, the message will instead be `Node.js version not specified, using 20.x`